### PR TITLE
feat: validation for community report

### PR DESCRIPTION
**Pull request by AntBush**
Implements # 1958

### CODE REVIEW

**Review by marcellmueller — COMMENTED** [review]
This is working mostly well though if I try and upload a community progress report with the wrong RFI number I get repeated errors that break the bottom of the page:

Screenshot 2023-08-15 at 7 25 19 AM

File: app/components/Analyst/Project/CommunityProgressReport/CommunityProgressReportForm.tsx [hunk 1]
```diff
@@ -106,7 +110,11 @@ const CommunityProgressReportForm = ({ application }) => {
   // will need to pass in something like setValidationErrors as a third argument in the validation ticket
```
Comment (marcellmueller) [inline app/components/Analyst/Project/CommunityProgressReport/CommunityProgressReportForm.tsx:110]: Can get rid of this outdated comment 🙂

**Comment by AntBush** [comment]
> This is working mostly well though if I try and upload a community progress report with the wrong RFI number I get repeated errors that break the bottom of the page:
> 
> Screenshot 2023-08-15 at 7 25 19 AM

I added in a check so it doesn't repeat the mismatch error if it's the same wrong ccbc number

**Review by marcellmueller — APPROVED** [review]
🥇

**Pull request merged**